### DISCUSSION TITLE
How to use this framework to make a simple dynamic interactive page?

### DISCUSSION THREAD

**Discussion by jackywu** [web-infra-dev/rspress#1544] [Q&A]
前端新手, 有一点点vue基础, 对react不了解. 已经使用rspress搭建了个人静态博客, 不过今天忽然发现需要一个简单的动态页面供交互, 逻辑也很简单, 该页面上提供1个input表单和提交按钮, post数据到后端api, 然后将结果显示在当前页面上. 请教一下, 这个在rspress里怎么实现呢? 感谢.

**Comment by Timeless0911** [comment]
Write a jsx component and import it in mdx.
